### PR TITLE
📝 [RUMF-1469] document new proxy configuration

### PR DESCRIPTION
**Pull request by BenoitZugmeyer**
### What does this PR do?

Update the RUM proxy configuration. Should be released after https://github.com/DataDog/browser-sdk/pull/1947

### Motivation

Context: https://docs.google.com/document/d/1XTl6iuCbzVnnNceMkvgJH-bsBdW7sP2ExnDKQzJDBF8/edit#heading=h.rnd972k0hiye

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by bcaudan — APPROVED** [review]
LGTM

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 1]
```diff
@@ -66,44 +66,36 @@ window.DD_RUM &&
 
 ## Proxy setup
 
-When your proxy receives data from the RUM Browser SDK, it must be forwarded to Datadog. The RUM Browser SDK adds the `ddforward` query parameter to all POST requests to your proxy. This query parameter contains the URL where all data must be forwarded to.
+When your proxy receives data from the RUM Browser SDK, it must be forwarded to Datadog. The RUM Browser SDK adds the `ddforward` query parameter to all POST requests to your proxy. This query parameter contains the URL path and parameters where all data must be forwarded to.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:69]:
```suggestion
When your proxy receives data from the RUM Browser SDK, it must be forwarded to Datadog. The RUM Browser SDK adds the `ddforward` query parameter to all POST requests to your proxy. This query parameter contains the URL path and parameters that all data must be forwarded to.
```

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 2]
```diff
@@ -66,44 +66,36 @@ window.DD_RUM &&
...
 To successfully proxy request to Datadog:
 
-1. Add a `X-Forwarded-For` header containing the request client IP address for accurate geoIP.
-2. Forward the request to the URL set in the `ddforward` query parameter using the POST method.
+1. Build the final Datadog intake URL by using:
+    - the Datadog intake origin corresponding to your site. See table below.
+    - the path and parameters provided by `ddforward`.
+2. Add a `X-Forwarded-For` header containing the request client IP address for accurate geoIP.
+3. Forward the request to the Datadog intake URL using the POST method.
 4. The request body must remain unchanged.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:78]:
```suggestion
4. Leave the request body unchanged.
```

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 3]
```diff
@@ -66,44 +66,36 @@ window.DD_RUM &&
...
 
-<div class="alert alert-danger">
-Ensure the <code>ddforward</code> attribute points to a valid Datadog endpoint for your <a href="https://docs.datadoghq.com/getting_started/site/" target="_blank">Datadog site</a>. <strong>Failure to do so may result in an insecure configuration.</strong>
-</div>
+The Site Parameter is an SDK [initialization parameter][1]. Datadog intake origins for each site are listed below:
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:80]:
```suggestion
The site parameter is an SDK [initialization parameter][1]. Datadog intake origins for each site are listed below:
```

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 4]
```diff
@@ -66,44 +66,36 @@ window.DD_RUM &&
...
-{{% tab "Latest version" %}}
-
-| Site    | Valid intake URL Pattern                       | Site Parameter      |
-|---------|------------------------------------------------|---------------------|
-| US1     | `https://*.browser-intake-datadoghq.com/*`     | `datadoghq.com`     |
-| US3     | `https://*.browser-intake-us3-datadoghq.com/*` | `us3.datadoghq.com` |
-| US5     | `https://*.browser-intake-us5-datadoghq.com/*` | `us5.datadoghq.com` |
-| EU1     | `https://*.browser-intake-datadoghq.eu/*`      | `datadoghq.eu`      |
-| US1-FED | `https://*.browser-intake-ddog-gov.com/*`      | `ddog-gov.com`      |
+The Datadog intake origin corresponding to your Site Parameter should be defined in your proxy implementation.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:90]:
```suggestion
The Datadog intake origin corresponding to your site parameter should be defined in your proxy implementation.
```

File: content/en/real_user_monitoring/guide/proxy-rum-data.md [hunk 5]
```diff
@@ -66,44 +66,36 @@ window.DD_RUM &&
...
-| US3     | `https://*.logs.us3-datadoghq.com/*` | `us3.datadoghq.com` |
-| EU1     | `https://*.logs.datadoghq.eu/*`      | `datadoghq.eu`      |
-| US1-FED | `https://*.logs.ddog-gov.com/*`      | `ddog-gov.com`      |
+Before Browser SDK v4.TODO, the `proxyUrl` initialization parameter was used and the Datadog intake origin was included in the <code>ddforward</code> attribute. The proxy implementation was in charge of validating this host and failure to do so resulted in various vulnerabilities.
 
-{{% /tab %}}
-{{< /tabs >}}
+The Datadog intake origin now needs to be defined in your proxy implementation to ensure security.
 
+<strong>If you are still using a proxy with an older version of the Browser SDK, please consider upgrading to a newer version of the Browser SDK.</strong>
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/proxy-rum-data.md:98]:
```suggestion
<strong>If you are still using a proxy with an older version of the Browser SDK, we recommend upgrading to a newer version of the Browser SDK.</strong>
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Left some feedback for your review. Otherwise, looks good!

**Review by rtrieu — APPROVED** [review]
Looks good, thanks!

**Pull request merged**